tests/fixtures/autouse_fixtures.py

```diff
@@ -118,8 +120,12 @@ def seed_all(rank_zero_seed: int, monkeypatch: pytest.MonkeyPatch):
 @pytest.fixture(autouse=True)
 def mapi_fixture(monkeypatch):
     # Composer auto-adds mosaicml logger when running on platform. Disable logging for tests.
-    mock_update = lambda *args, **kwargs: None
+    future_obj = Future()
+    future_obj.set_result(None)
+    mock_update = lambda *args, **kwargs: future_obj
     monkeypatch.setattr(mcli, 'update_run_metadata', mock_update)
+    # Disable the MosaicMLLogger by default
+    monkeypatch.setenv(MOSAICML_PLATFORM_ENV_VAR, 'false')
```

Comment:
Because of this, don't we need to explicitly turn it back on in `test_mosaicml_logger.py`?

Reply:
we can remove I don't think we need this either.